omniflixhub/assetlist.json

```diff
@@ -29,6 +29,243 @@
+          },
+          "chain": {
+            "channel_id": "channel-30",
+            "path": "transfer/channel-30/uist"
+          }
+        }
+      ],
+      "images": [
+        {
+          "image_sync": {
+            "chain_name": "agoric",
+            "base_denom": "uist"
+          },
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/agoric/images/ist.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/agoric/images/ist.svg"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/agoric/images/ist.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/agoric/images/ist.svg"
+      }
+    },
+    {
+      "denom_units": [
+        {
+          "denom": "ibc/AC4C84DD7D3732CFA95A420247FAD5179BE4F42E5635E2BF0A62C67E2E2FA4A9",
+          "exponent": 0,
+          "aliases": [
+            "uusdc"
+          ]
+        },
+        {
+          "denom": "usdc",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/AC4C84DD7D3732CFA95A420247FAD5179BE4F42E5635E2BF0A62C67E2E2FA4A9",
+      "name": "USD Coin (Axelar)",
+      "display": "usdc",
+      "symbol": "USDC.axl",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "axelar",
+            "base_denom": "uusdc",
+            "channel_id": "channel-77"
+          },
+          "chain": {
+            "channel_id": "channel-27",
+            "path": "transfer/channel-27/uusdc"
+          }
+        }
+      ],
+      "images": [
+        {
+          "image_sync": {
+            "chain_name": "axelar",
+            "base_denom": "uusdc"
+          },
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/axelar/images/usdc.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/axelar/images/usdc.svg"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/axelar/images/usdc.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/axelar/images/usdc.svg"
+      }
+    },
+    {
+      "denom_units": [
+        {
+          "denom": "ibc/9F28238BDB687A86681B30764BA1437E079C3565D04FDD7862055A83F7E5C630",
+          "exponent": 0
+        },
+        {
+          "denom": "baddog",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/9F28238BDB687A86681B30764BA1437E079C3565D04FDD7862055A83F7E5C630",
+      "name": "BadDog (Osmosis)",
+      "display": "baddog",
+      "symbol": "BADDOG.osmo",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "osmosis",
+            "base_denom": "factory/chihuahua1x4q2vkrz4dfgd9hcw0p5m2f2nuv2uqmt9xr8k2/achihuahuawifhat",
```

Comment:
The base denom is for BADDOG once transferred to Osmosis is: ibc/2FFE07C4B4EFC0DDA099A16C6AF3C9CCA653CC56077E87217A585D48794B0BC7

Reply:
as per: https://github.com/cosmos/chain-registry/blob/820c602410c8126c5920269dff3b1290f64e09b0/osmosis/assetlist.json#L13283

Reply:
okay understood now
updated correct base denom